contracts/ServiceNodeContribution.sol

```diff
@@ -404,7 +484,7 @@ contract ServiceNodeContribution is Shared {
      */
     function minimumContribution() public view returns (uint256 result) {
         result = calcMinimumContribution(
-            stakingRequirement - totalContribution(),
+            stakingRequirement - totalContribution() - totalReservedContribution(),
             contributorAddresses.length,
```

Comment:
This `numContributors` parameter value here needs to include unfilled, but reserved addresses as well.